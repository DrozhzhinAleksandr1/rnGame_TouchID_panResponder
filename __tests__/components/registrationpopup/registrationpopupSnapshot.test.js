import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { RegistrationPopUp } from '../../../app/components/registrationpopup/registrationpopup';

it('create snapshot RegistrationPopUp', () => {
    const renderer = new ShallowRenderer()
    let snapshot = renderer.render(<RegistrationPopUp showRegistrationPopUp={false} />);
    expect(snapshot).toMatchSnapshot();
});
it('create snapshot RegistrationPopUp', () => {
    const renderer = new ShallowRenderer()
    let snapshot = renderer.render(<RegistrationPopUp showRegistrationPopUp={true} />);
    expect(snapshot).toMatchSnapshot();
});
