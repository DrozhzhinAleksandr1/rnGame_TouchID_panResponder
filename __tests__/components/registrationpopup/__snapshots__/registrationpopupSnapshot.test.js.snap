// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot RegistrationPopUp 1`] = `
<TouchableWithoutFeedback
  accessibilityLabel="BtnShowToolbarDisablerID"
  onPress={[Function]}
  testID="BtnShowToolbarDisablerID"
>
  <Component
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0)",
        "bottom": 40,
        "elevation": 10,
        "maxHeight": 58,
        "minHeight": 58,
        "position": "absolute",
        "width": "100%",
        "zIndex": 10,
      }
    }
  />
</TouchableWithoutFeedback>
`;

exports[`create snapshot RegistrationPopUp 2`] = `null`;
